cpu/native/periph/timer.c

```diff
@@ -47,7 +47,10 @@
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
-#define NATIVE_TIMER_SPEED 1000000
+#define NATIVE_TIMER_SPEED          (1000000)
+#define NATIVE_TIMER_DEFAULT_SCALE  (0)
```

Comment:
Please add a sentence or two explaining what ```native_timer_scale``` is used for.

Reply:
Done